modules/core/kivent_core/systems/gameview.pyx

```diff
@@ -1,14 +1,15 @@
 # cython: embedsignature=True
 from gamesystem cimport GameSystem
 from kivy.properties import (StringProperty, ListProperty,
-    NumericProperty, BooleanProperty, ObjectProperty)
+    NumericProperty, BooleanProperty, ObjectProperty, AliasProperty)
 from kivy.clock import Clock
 from kivy.vector import Vector
 from kivent_core.managers.system_manager cimport SystemManager
 from kivy.graphics.transformation import Matrix
 from kivy.graphics import RenderContext
 from kivy.factory import Factory
 from kivy.input import MotionEvent
+from math import sin, cos

```

Comment:
We should be using the libc sin and cos instead of python:
For an example of what the import would look like:
https://github.com/kivy/kivent/blob/863291a87a4cb5375d5572bcd3759e7f298b301d/modules/particles/kivent_particles/particle_math.pxi#L3